libdnf/rpm/repo.cpp

```diff
@@ -223,7 +223,7 @@ const std::string & Repo::Impl::get_metadata_path(const std::string & metadata_t
     }
     auto & ret = (it != metadata_paths.end()) ? it->second : empty;
     if (ret.empty()) {
-        logger.debug(fmt::format("not found \"{}\" for: {}", metadata_type, conf->name().get_value()));
+        logger.debug(fmt::format("not found \"{}\" for: {}", metadata_type, id));
```

Comment:
Why did you change to report repoid instead of name? I think that repoid is a better better choice because it is unique but still I have an interest to know your reasons.

Reply:
Exactly. The repository id is unique "name" of the repository. And it is used in libsolv too.
But what is `conf->name()`? A nicer more descriptive name. I think it is good for output to console. 
For logging I prefer unique repository id.